.buildkite/scripts/steps/serverless-integration-tests.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+# This script runs the serverless integration tests
+set -eo pipefail
+
+BUILDKITE=${BUILDKITE:-}
+
+# Check if the required binary is installed
+if ! command -v oblt-cli &> /dev/null; then
+    echo "oblt-cli is not installed. Please install it to run this script."
+    exit 1
+fi
+
+if [ -n "$BUILDKITE" ]; then
+    buildkite-agent artifact download "build/distributions/**" . "$BUILDKITE_BUILD_ID"
+fi
+
+if [ -z "$BUILDKITE" ]; then
+    echo "~~~ Creating the cluster if no CI environment is detected"
+    # TODO name of the cluster should be set dynamically to avoid conflicts when local testing.
+    cluster_name=$(.buildkite/scripts/steps/provision-cluster.sh "ea-int-test-local")
+else
+    # Use the day of the week to get a unique cluster name for each day
+    # The provision happens somewhere else.
+    cluster_name="ea-int-test-$(date +%a | tr '[:upper:]' '[:lower:]')"
+fi
+
+echo "~~~ Retrieving the credentials for the cluster"
+oblt-cli cluster secrets env --cluster-name="${cluster_name}" --output-file="${PWD}/env.sh"
+
+source "${PWD}/env.sh" && rm -f "${PWD}/env.sh"
+
+echo "+++ Run integration-tests"
+# TBC
```

Comment:
@pkoutsovasilis, do you mind if I ask you to help with what's the command I need to enable here to run the serverless ITs? And also what's the env variable contract for using the existing created project?

Bear in mind, this is still in development, but I guess you can see a bit what the skeleton is to facilitate running the same script locally and in the CI. It won't install the binary locally for now, the CI uses https://github.com/elastic/oblt-cli-buildkite-plugin/

Reply:
I see some references about 

```
.buildkite/scripts/steps/integration_tests.sh serverless integration:single TestLogIngestionFleetManaged #right now, run a single test in serverless mode as a sort of smoke test, instead of re-running the entire suite
```

but I'm not much familiar with how that works, so I pretty much want to know the name of the env variables I need to pass to the serverless ITs and the mage command to run

Reply:
ok I will prepare but keep the changes local then and when you are ready to try and invoke a CI ping me and I will push them here 🙂

Reply:
@pkoutsovasilis , do you want to contribute to this PR by adding the mage commands you want?

Reply:
we don't need this script (or the ps1 equivalent) anymore 🙂